components/dropdown/dropdown-menu.js

```diff
@@ -18,6 +18,7 @@ class DropdownMenu extends ThemeMixin(DropdownContentMixin(LitElement)) {
 
 	constructor() {
 		super();
+		this.__applyFocus = true;
```

Comment:
Isn't `__applyFocus` in the base mixin `true` by default anyway?

Reply:
Should dropdown menu always be used with a component that uses the `opener-mixin`? We have a couple of examples for dropdown that just use `button` and this variable is unset for those elements, I was thinking that could be a viable use case. If we don't want to support that then setting this through the mixin should be enough

Reply:
Maybe I'm missing something. Searching through `dropdown-menu.js`, this is the only occurrence of `__applyFocus`.  It's defined in `DropdownContentMixin` (which this class extends) but in there it's also set to `true` in its constructor. So I don't see what setting it to `true` in the constructor here is doing that isn't already being done in the mixin constructor.

Reply:
Yeah this was dumb, one of the `toggleOpen` calls was passing `undefined`  instead of `true` making me think it wasn't getting set properly for some reason. Changing the toggle call makes this unnecessary